Makefile

```diff
@@ -156,9 +156,11 @@ endif
 $(PROTO_PY_OUTS): py/vtproto/%_pb2.py: proto/%.proto
 	$(PROTOC_COMMAND) -Iproto $< --python_out=py/vtproto --grpc_python_out=py/vtproto
 
+# TODO(sougou): find a better way around this temp hack.
+VTTOP=$(VTROOT)/../../..
```

Comment:
In the contributor's guide, we are asking people to checkout to `~/vitess`. This will break anyone who follows that.
What error are you getting from `make proto`?

Reply:
Run as is, you'll get this error:
```
~/...vitess> make proto
Tue Dec 31 12:30:45 PST 2019: Compiling proto definitions
/bin/bash: line 1: cd: /home/sougou/dev/src/vitess.io/vitess/src: No such file or directory
[more of the same]
```

If you fix the `cd` command, then you'll get something like this:
```
~/...vitess> make proto
Tue Dec 31 12:32:38 PST 2019: Compiling proto definitions
vitess.io/vitess/proto: warning: directory does not exist.
vitess.io/vitess/proto/tabletmanagerservice.proto: No such file or directory
```

The .proto files, there is this line:
```
option go_package = "vitess.io/vitess/go/vt/proto/tabletmanagerdata";
```
This makes protoc want to generate the file at `vitess.io/vitess/proto` relative to the current path. We need to figure out a way to make it generate the file under `vt/proto` instead. I haven't found a way yet.

If we change the go_package to `go/vt/proto/tabletmanagerdata`, then it generates the wrong package names in the generated files (without thevitess.io/vitess prefix).

Essentially, protos won't build with the current Makefile at all. This change at least allows you to build it if you have vitess checked out under `src/vitess.io/vitess`.

Reply:
Could we potentially use a symlink from `vitess.io/vitess/proto` to `vt/proto` instead? .. at least until we find a better way to do this.